packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -800,6 +808,7 @@ class DropdownButton<T> extends StatefulWidget {
     this.focusColor,
     this.focusNode,
     this.autofocus = false,
+    this.dropdownColor,
```

Comment:
Somewhere in the DropdownButton doc string should mention how and when to use dropdownColor